src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -497,4 +497,7 @@
         Items="@(CorrelationFile)"
         Overwrite="true" />
   </Target>
+
+  <!-- Setup Dumpling service to collect and upload any crash dumps -->
+  <Import Project="$(MSBuildThisFileDirectory)Dumpling.targets" />
```

Comment:
Do we actually want to turn this on for every test run ever?  (Which is what this is doing).  I was under the impression there'd be a property like "EnableDumpling".

I also thought it was a goal to turn this on for local non-Helix runs, in which case this would need to live elsewhere to get imported when testing locally.